src/Integrations/Modal.js

```diff
@@ -0,0 +1,27 @@
+import { useRef } from 'react'
```

Comment:
Why is there a Modal and a Gallery? They kind of look like they're doing the same thing. Can you screenshot me the difference?

Reply:
![screenshot homepage](https://user-images.githubusercontent.com/47989605/109340161-807d1d80-781d-11eb-83b9-85bba8efd73b.jpg)

The modal is if you click onto the gallery pic, which the picture should transform when hovered.

![screenshot modal](https://user-images.githubusercontent.com/47989605/109340183-8a068580-781d-11eb-8011-32bdb8f20b24.png)